src/components/tech/WorkingBox.tsx

```diff
@@ -0,0 +1,75 @@
+import {
+  Badge,
+  Box,
+  Container,
+  Flex,
+  Heading,
+  Link,
+  Section,
+  Text,
+} from "@radix-ui/themes";
+import React from "react";
+
+interface WorkingBoxProps {
+  title: string;
+  para: string;
+  link: string;
+  tech: string;
+}
+
+const colors: { [key: number]: string } = {
+  0: "#3E63DD",
+  1: "#5AC597",
+  2: "#98BEC6",
+  3: "#DFCDE8",
+  4: "#4362A6",
+};
+
+export const WorkingBox = ({ title, para, link, tech }: WorkingBoxProps) => {
+  const data = tech.split(",");
+  return (
+    <Flex
+      gap="4"
+      align="start"
+      direction="column"
+      p="20px"
```

Comment:
**question (non-blocking):** Should we just use [Radix spacing](https://www.radix-ui.com/themes/docs/theme/spacing) size 4 (16px) or size 5 (24px) here, instead of a specific px value?